fix(loop-unroll): must ignore disable flag in ir

priyasiddharth opened the pull request:
clang-14 can add llvm.loop.unroll.disable flag at times. See bitcode of mcfuzz/issue_48.c for an example. Seahorn should make decision to unroll independently of what LLVM/Clang thinks.

Also small formatting fixes done by editor

priyasiddharth commented:
@agurfinkel PTAL

agurfinkel reviewed (approved):
It would be better if the editing changes are undone.

This file is created by merging with changes in LLVM. The fewer syntactic changes there are, the easier it is to remember what was the original change and resolve any future merge conflicts.

Otherwise, this looks great. Please merge in.

priyasiddharth committed:
fix(loop-unroll): must ignore disable flag in ir

clang-14 can add llvm.loop.unroll.disable flag at times. See
bitcode of mcfuzz/issue_48.c for an example. Seahorn should
make decision to unroll independently of what LLVM/Clang thinks.

priyasiddharth commented:
I've removed changes due to auto reformatting. @agurfinkel  You will have to merge since I don't have write access to `llvm-seahorn`.
